cst438_register/src/main/java/com/cst438/domain/StudentDTO.java

```diff
@@ -6,13 +6,15 @@ public class StudentDTO {
 	public String email;
 	public int statusCode;
 	public String status;
+	public String message;
 	
 	public StudentDTO() {
 		this.student_id = 0;
 		this.name = null;
 		this.email = null;
 		this.status = null;
 		this.statusCode = 0;
+		this.message = "";

```

Comment:
Interesting.  Wouldn't the status code and the message be added on via the Spring Boot framework?  I wonder if the DTO is just the data itself, outside of the response details.

Reply:
The status code is a field of the Student table. It represents a hold on a student's account. Status code == 0 means all clear, StatusCode !=0 means a hold is present, and Status is a string that would then read something like "Payment overdue" or "Alcohol Awareness". As far as the message unit yes, that could technically be slapped into the HTTP response although maybe a meaningless change at the moment.